src/sentry/api/endpoints/organization_tagkey_values.py

```diff
@@ -22,7 +22,16 @@ def get(self, request, organization, key):
         except OrganizationEventsError as exc:
             return Response({'detail': exc.message}, status=400)
 
-        query = 'tags_key:%s' % (key,)
+        # release is a weird tag
+        if key == 'release':
+            key = 'sentry:release'
+
+        is_column = key in SENTRY_SNUBA_MAP
+
+        if is_column:
+            query = 'has:"%s"' % (key,)
```

Comment:
this fails if the column type is an array (e.g. `stack.filename`)

Reply:
i _think_ it should be fixed by this: https://github.com/getsentry/sentry/pull/10663

and alex is also working on a better solution that fixes from snuba (so we don't have to worry about types)